### PR TITLE
docs: add zero-value emphasis on 'movementX' and 'movementY'

### LINKED ISSUE #29315: Document that MouseEvent.movementX/movementY are 0 for events other than `mousemove`
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/movementX

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

> movementX/movementY must be zero for all mouse events except mousemove.

https://w3c.github.io/pointerlock/#dom-mouseevent-movementx

This should be documented as I tried to use it for `mouseenter`.

### What did you expect to see?

"movementX/movementY must be zero for all mouse events except `mousemove`."

### Do you have any supporting links, references, or citations?

https://w3c.github.io/pointerlock/#dom-mouseevent-movementx

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/mouseevent/movementx`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/movementX
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/mouseevent/movementx/index.md
* Last commit: https://github.com/mdn/content/commit/acfe8c9f1f4145f77653a2bc64a9744b001358dc
* Document last modified: 2023-07-07T07:19:19.000Z

</details>

### PR DESCRIPTION
**Pull request by Namchee**
### Description

This pull request adds information about zero-value on `movementX` and `movementY`.

According the w3c specification below, the value of `movementX` and `movementY` will always be zero except for `mousemove` event.

### Motivation

This pull request adds an emphasis on the value of `movementX` and `movementY` that will always be zero on any other `MouseEvent` other than `mousemove`, therefore preventing logical errors when readers are dealing with `MouseEvent`.

### Additional details

[Specification on movementX](https://w3c.github.io/pointerlock/#dom-mouseevent-movementx)

### Related issues and pull requests

Fixes #29315

### CODE REVIEW

File: files/en-us/web/api/mouseevent/movementx/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ In other words, the value of the property is computed like this: `currentEvent.m
 
 ## Value
 
-A number.
+A number. Always zero on all {{domxref("MouseEvent")}} other than `mousemove`.
```
Comment (sideshowbarker) [inline files/en-us/web/api/mouseevent/movementx/index.md:18]:
```suggestion
A number. Always zero on any {{domxref("MouseEvent")}} other than `mousemove`.
```

File: files/en-us/web/api/mouseevent/movementy/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ In other words, the value of the property is computed like this: `currentEvent.m
 
 ## Value
 
-A number.
+A number. Always zero on all {{domxref("MouseEvent")}} other than `mousemove`.
```
Comment (sideshowbarker) [inline files/en-us/web/api/mouseevent/movementy/index.md:18]:
```suggestion
A number. Always zero on any {{domxref("MouseEvent")}} other than `mousemove`.
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Cristopher, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Comment by sideshowbarker** [comment]
> Cristopher, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

Sorry, I see now I should have said, “Namchee…”

**Comment by Namchee** [comment]
> Sorry, I see now I should have said, “Namchee…”

Nah it's alright.

Sorry for not being able to fix the issues on time.

**Pull request merged**